pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -620,6 +621,20 @@ def _render_jira_template(fields: Dict, template_vars: Dict):
             _LOGGER.warning("Failed to render JIRA template text: %s", ex)
         return fields
 
+    def set_advisory_dependencies(self, advisories):
+        target_kind = ['rpm', 'metadata']
+        expected_blocking = {advisories['image'], advisories['metadata']}
```

Comment:
A certain advisory might not exist. Would like to skip that dependency if one of the advisory kind doesn't exist.

Reply:
Added